travis-scripts/before_install_osx.sh

```diff
@@ -54,21 +55,19 @@ sudo installer -pkg /Volumes/XQuartz-*/XQuartz.pkg -target /
 popd 
 
 echo "doing an OSX before install step."
-# python may require a little force to install. In October 2014
-# there were issues with easy_install and pip not letting the install
```

Comment:
These were fixed a while ago. Doesn't need to be kicked anymore. You may wish to double check, but should be :+1:.